Runtime/ExitPoll/ExitPoll.cs

```diff
@@ -201,6 +203,48 @@ public void BeginExitPoll(ExitPollParameters parameters)
             }
         }
 
+        /// <summary>
+        /// when you manually need to close the Exit Poll question set manually OR
+        /// when requesting a new exit poll question set when one is already active
+        /// </summary>
+        public void EndQuestionSet(int timeToWait)
+        {
+            panelProperties.Clear();
+            if (CurrentExitPollPanel != null)
+            {
+                CurrentExitPollPanel.CloseError(timeToWait);
+            }
+            OnPanelError();
+        }
+
+        private void Cognitive3D_Manager_OnUpdate(float deltaTime)
```

Comment:
If controllers and/or hands are unavailable for over a certain time, fallback to HMD